package.json

```diff
@@ -8,13 +8,11 @@
     "node": ">=6.0.0"
   },
   "repository": "googleapis/nodejs-bigtable",
-  "main": "./src/index.js",
+  "main": "build/src/index.js",
   "files": [
-    "protos",
-    "src",
-    "AUTHORS",
-    "CONTRIBUTORS",
-    "LICENSE"
+    "build/protos",
+    "build/proto",
```

Comment:
Is this going to be an iterative process like the other repos? We might want to ignore the declaration/map files for now.

Reply:
Yes, and good call.  Updated!